process_jartic_archives_fast.py

```
@@ -0,0 +1,217 @@
+                
+                batch_progress.update(1)
+                
+                # Flush periodically
+                if self.total_measurements % 100000 == 0:
+                    f.flush()
+        
+        batch_progress.close()
+        print(f"✅ Completed {month_str}: {self.total_measurements:,} measurements")
+
+
+async def main(args):
```

Ruff (0.12.2)

`main` is too complex (12 > 10)

(C901)